client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -218,7 +214,7 @@ export const AppSchemaForm = ({ app }) => {
                 History &gt; Jobs
               </Link>
               .
-            </Alert>
+            </Message>
```

Comment:
See earlier comment for opening tag.

```suggestion
            </SectionMessage>
```